picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java

```diff
@@ -108,7 +105,8 @@ public void run() {
     pinStarBlueSymbol.setHeight(40);
     pinStarBlueSymbol.setWidth(40);
     //Optionally set the offset, to align the base of the symbol aligns with the point geometry
-    pinStarBlueSymbol.setOffsetY(11); //The image used for the symbol has a transparent buffer around it, so the offset is not simply height/2
+    pinStarBlueSymbol.setOffsetY(
+        11); //The image used for the symbol has a transparent buffer around it, so the offset is not simply height/2
```

Comment:
@willcrick Can you move this comment above the method call, then you won't need to return and reads better?